x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_params_schema.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { signalParamsSchema, SignalParamsSchema } from './signal_params_schema';
+import {
+  getSignalParamsSchemaDecodedMock,
+  getSignalParamsSchemaMock,
+} from './signal_params_schema.mock';
+import { DEFAULT_MAX_SIGNALS } from '../../../../common/constants';
+
+describe('signal_params_schema', () => {
+  test('it works with expected basic mock data set', () => {
+    const schema = signalParamsSchema();
+    expect(schema.validate(getSignalParamsSchemaMock())).toEqual(
+      getSignalParamsSchemaDecodedMock()
+    );
+  });
+
+  test('it works on older lists data structures if they exist as an empty array', () => {
```

Comment:
Is this now a value we'll need to support indefinitely? Or a workaround until lists are in prod?

Reply:
It is a value we need to support until we write migration hooks within the saved objects (which we haven't done just yet).